examples/python_bindings.py

```diff
@@ -0,0 +1,19 @@
+from rattler import Package, Channel, Environment
```

Comment:
Would it make sense to move this into the `py-rattler/examples` folder? There are already 3 examples there anyway. Might be nice to link those in the readme as well?